src/utils/poll.jl

```diff
@@ -0,0 +1,56 @@
+struct WatchState
+    filename::String
+    mtime ::Float64
+    WatchState(filename) = new(filename, mtime(filename))
+end
+function poll_until_changed(files::Set{String}; interval = 1.)
+    watched = WatchState[]
+    for f in files
+        !isfile(f) && return
+        push!(watched, WatchState(f))
+    end
+    active = true
+    while active
+        for w in watched
+            if !isfile(w.filename) 
+                active = false
+                break;
+            end
+            if mtime(w.filename) != w.mtime
+                active = false
+                break;
+            end
+            sleep(0.1)
```

Comment:
@waralex Is this to ensure a minimum amount of time passes regardless of the value of `interval`?

Reply:
@rpkyle Yes, I overdid it with this timeout, I will remove it.

Reply:
@rpkyle fixed at 556f838